README.md

```diff
@@ -1,2 +1,97 @@
-# School-Library
-Creating School Library Website, Using Ruby as for the Back end 
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>School Library</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 MORSE CODE <a name="about-project"></a>
+
+> **[School-Library]** is an educational project. This is used to learn Ruby methods. 
+
+## 🛠 Built With <a name="built-with">Ruby</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Ruby
+
+
+### Key Features <a name="key-features"></a>
+
+- **Use Ruby class**
+- **Use Ruby Methods**
+- **Use any?, all? and filter? to solve problems**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Prerequisites
+
+In order to run this project you need: VScode and Ruby Installed
+
+### Setup
+
+Clone this repository to your desired folder:
+
+run: `git clone https://github.com/shakerAbuDrais/School-Library.git`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@Shaker](https://github.com/shakerAbuDrais/)
+- LinkedIn: [Shaker](https://www.linkedin.com/in/shaker-abady/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/shakerAbuDrais/School-Library.git/issues).
```

Comment:
Great job @shakerAbuDrais 🤝

- [ ] [Optional] Kindly fix the issues link to make your readme file more professional.